.github/workflows/check-release.yml

```diff
@@ -0,0 +1,117 @@
+name: Check versions and build-publish
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install requests
+          pip install --upgrade hatch
+
+      - name: Check which 📦 needs to be released
+        run: |
+          packages=("vizro-core") #  "vizro-core"
+
+          for package in "${packages[@]}"; do
```

Comment:
You've prepared this configuration so it can handle a process with multiple packages. Do we already have the potential candidate that's not `vizro-core` and did you test it with multiple packages?

Reply:
Yes, the work of AX will be the next package, and I tested it with a hypothetical next package, but I expect that there will still be slight adjustments necessary when it comes to that